ECMA2Yaml/ECMAHelper/ECMALoader.docs.cs

```diff
@@ -121,6 +123,54 @@ public Docs LoadDocs(XElement dElement)
             };
         }
 
+        /// <summary>Downgrades markdown headers from 1 - 5. So a `#` becomes `##`, but `######` (ie. h6) remains the same.</summary>
+        /// <param name="remarksText">A string of markdown content</param>
+        public static string DowngradeMarkdownHeaders(string remarksText)
+        {
```

Comment:
the original template logic only triggers if `<h2>` is detected in remarks, otherwise everything remain untouched. We should follow this to keep compatibility.

Reply:
The code will now only trigger if it finds an H2 in the content